boards/earlgrey-nexysvideo/Cargo.toml

```diff
@@ -28,3 +28,6 @@ lowrisc = { path = "../../chips/lowrisc" }
 #      OpenTitan SoC design simulated in Verilator.
 fpga_nexysvideo = ["earlgrey/config_fpga_nexysvideo"]
 sim_verilator = ["earlgrey/config_sim_verilator"]
+# This is used to indicate that we should include the prebuilt
+# RISC-V assmebly. This is currently just used to run RISC-V tests
+asm = []
```

Comment:
This is a bit of a generic name, "asm" could have many meanings and side-effects. Could we call this something more descriptive?